src/frontend/apps/standalone_site/src/routes/routes.tsx

```diff
@@ -238,6 +238,9 @@ export const routes: Routes = {
           UPDATE: {
             path: `/my-contents/videos/:videoId`,
           },
+          DELETE: {
+            path: `/my-contents/videos/delete`,
+          },
         },
```

Comment:
You shouldn't create a route for `delete`. 
A route was created for `create` to have a quick access for creating a resource, but for delete it is different because you need to check the resources that you want to delete in order to have your modal working.
If I type in my browser `http://localhost:3000/my-contents/videos/delete` the modal open but nothing is deletable.

Reply:
What shoulf I do instead then ? I thought I had to do it this way because the backend uses the same route to bulk delete resources

Reply:
Ok I think this is what I did at first but for some reason it didn't work as expected. I should try again then thanks

Reply:
I tested again and I get now why I had to do it with a route. The modal doesn't appear when managing it with a state, and this is the reason why we open the creation modal with a route as well. I don't know how to solve this issue in another way than keeping the `delete` route